chore: add logs information to developer tips (#5865)
iotaledger/firefly: docs/guides/developer-tips.md

Firefly uses [wallet.rs](https://github.com/iotaledger/wallet.rs) in the backend to handle functionality around value-based transfers.
See its [README](https://github.com/iotaledger/wallet.rs#dependencies) for the required dependencies.

> The wallet.rs team often requires logs to debug their issues. Currently, they can be found in the following folders:

Reviewer: ```suggestion
Log files are often required to debug `wallet.rs` issues. They can be found in the following folders:
```
